name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/csv/CSVPortfolioExtractor.java

```diff
@@ -35,7 +35,7 @@
         fields.add(new Field("time", Messages.CSVColumn_Time).setOptional(true)); //$NON-NLS-1$
 
         fields.add(new ISINField("isin", Messages.CSVColumn_ISIN).setOptional(true)); //$NON-NLS-1$
-        fields.add(new Field("ticker", Messages.CSVColumn_TickerSymbol).setOptional(true)); //$NON-NLS-1$
```

Comment:
This is a breaking change for existing users. The "code" used in the JSON configurations that the user can create (and are stored locally). I do not think users are editing or seeing the JSON, therefore I tend to say: let's not migrate the configurations but rather leave the code "ticker" here.